### PR TITLE
capture logging messages

### PR DESCRIPTION
**Pull request by matthiasdiener**
Modeled after the warnings capture.

### CODE REVIEW

**Comment by matthiasdiener** [comment]
This is ready for a first look @inducer

File: logpyle/__init__.py [hunk 1]
```diff
@@ -385,7 +388,18 @@ def _set_up_schema(db_conn: Connection) -> int:
         lineno integer
         )""")
 
-    schema_version = 2
+    # schema_version < 3 does not have the logging table
+    db_conn.execute("""
+      create table logging (
+        rank integer,
```
Comment (inducer) [inline logpyle/__init__.py:394]: May want to store wall time? (Possibly also for warnings?)

**Review by inducer — COMMENTED** [review]
Thanks! Other than the one suggestion, I don't see any issues here. Do we have a way to upgrade between schema versions? If not, this may be easier than maintaining code checking for older schemas.

**Comment by inducer** [comment]
> Do you mean, adding a script that converts an older schema version db to a new one?

Either that (plus raising an error mentioning that script when trying to open a file with an obsolete schema), or converting the schema "behind the user's back" on open.

File: logpyle/__init__.py (hunk 1 above)
Comment (matthiasdiener) [reply logpyle/__init__.py:395]: I added a unixtime column in [97add34](https://github.com/illinois-ceesd/logpyle/pull/94/commits/97add3405575f11dafd38f5a6ac53475ce2e0213) to both tables.

**Comment by matthiasdiener** [comment]
> > Do you mean, adding a script that converts an older schema version db to a new one?
> 
> Either that (plus raising an error mentioning that script when trying to open a file with an obsolete schema), or converting the schema "behind the user's back" on open.

I added this as a new issue (#99). I verified that with this PR, older databases can still be opened and work as expected (without some columns and without the logging table). I think this is ready for review.

**Pull request merged**